Module 1:Portfolio popup menu

helina42 opened the pull request:
This project includes:

1. Through a creating a popup() function and linking through ID the following is done. 
2. Through the toggle element the menu options appear and disappear when needed.

- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

helina42 committed:
Add Html-css-javascript linter setup

helina42 committed:
Add pop-up menu to portfolio

Adams-Ghub committed:
Add menu options to popmenu

helina42 committed:
Update linters.yml

mikethreels commented on style.css:
```diff
@@ -15,6 +23,48 @@ body {
+  font-weight: 600px;
+  font-size: 32px;
+  line-height: 44px;
+}
```
- [x] the menu pop-up should span the entire width of the screen as shown in the template:
- [ ] the background should be blurred as well when the pop-up is open.
your project:
![afbeelding](https://user-images.githubusercontent.com/39240820/224758950-72ad197f-7b1d-42b8-b855-011d907484a4.png)
figma:
![afbeelding](https://user-images.githubusercontent.com/39240820/224759108-01b337a1-a774-487e-8ca2-cc057e14596b.png)

mikethreels reviewed (changes requested):
Hi team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

helina42 committed:
Add width to popup after review

helina42 commented:
@mikethreels  hi 
 Thank you for the helpful review 
I have fixed the first problem that you have mentioned by fixing the width of the pop up.

On the second issue you have mentioned, as you can see from the code I have added  backdrop-filter:blur(6px); on line 37 and it seems to be working fine on mine and on my partners computer and as you can see from the snapshot of my portfolio so I don't understand what exactly I should fix. 

![Capture](https://user-images.githubusercontent.com/112048040/224772006-e28a182a-32ee-4bed-b1a0-3046ac334d7e.PNG)

jasonfelice commented on style.css:
```diff
@@ -15,6 +23,48 @@ body {
   white-space: nowrap;
 }
 
+.menu-popup {
```
- [x] This is a nav menu popup window, so it should be fixed in the viewport. Otherwise, the user has to scroll to this menu.
- [x] You have hard-coded many attributes which are resulting in inconsistent responsiveness,
- You can update this entire CSS to the following:
```
    background-color: #6070ff;
    position: fixed;
    top: 0;
    bottom: 0;
    right: 0;
    left: 0;
    display: flex;
    flex-direction: column;
    backdrop-filter: blur(6px);
    align-items: start;
    font-weight: 600px;
    font-size: 32px;
    line-height: 44px;
```

helina42 replied:
@jasonfelice hi, 
Thank you so much for your review!
As per your review I have made changes to my code. Could you please re-review?

jasonfelice reviewed (changes requested):
Hi @helina42, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
No Linter errors✔️
Followed GitHub Flow✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

helina42 committed:
Add styles to popup after review

HAMMAS-SALEEM commented on style.css:
```diff
@@ -26,18 +26,18 @@ body {
+  font-weight: bold;
   font-size: 32px;
   line-height: 44px;
 }
```
## Problem
![image](https://user-images.githubusercontent.com/93039359/224918127-9cb20f36-5d2e-43e2-b600-0b32f76f6b5c.png)


- [ ] Great work on implementing some of the changes requested by the previous reviewer 👍 it would be nice if you fix your menu responsiveness issue as suggested by the previous reviewer 😊 I saw that you are trying to fix it using your CSS well I fixed the responsiveness issue. You can also fix it by following the steps below ⬇️

## First Step 
Kindly remove the `width: 100%;` from **.header**

![image](https://user-images.githubusercontent.com/93039359/224917233-84aed94f-f577-4cc9-9721-10a6f913ebc6.png)

## Second Step
Kindly remove `min-width: 375px` from **.middle-section1** & **.middle-section2**

![image](https://user-images.githubusercontent.com/93039359/224917543-cf1b5041-f17f-4b91-a372-346a961719d3.png)

![image](https://user-images.githubusercontent.com/93039359/224917790-b9dfb88a-8a87-4c6e-8aca-cfbaa9d40962.png)

## Last Step
Add `background-size: cover;` in **.menu-popup**

![image](https://user-images.githubusercontent.com/93039359/224920143-e6de06d5-f66f-4754-a208-455684bf0b4a.png)

HAMMAS-SALEEM commented on index.html:
```diff
@@ -10,13 +10,30 @@
+      <div onclick="popmenu()" class="close-mobile-menu" id="close-mobile-menu">
+        <img src="./Icons/Icon - Cancel.png" alt="close menu button"/>
+      </div>   
+    <div class="menu-li">
+        <nav><a href="#Portfolio" onclick="popmenu()">Portfolio</a></nav>
```
![image](https://user-images.githubusercontent.com/93039359/224921529-65291d82-0270-4682-ba6e-d45bff3e5c16.png)

**[OPTIONAL]**


- [ ] KIndly fix this button because it is not taking us to the right section of your page 😨 It is taking us to your homepage but it should take us to the cards which you have by the class name of **middle-section1 & middle-section2** 😊 Kindly fix this issue to make your project professional 👍

HAMMAS-SALEEM reviewed (changes requested):
# Required Changes :recycle:
![almost-there](https://user-images.githubusercontent.com/65860678/170491251-b1add8ba-b18f-4648-a893-20e510d957fd.gif)

## Hey @helina42, @Adams-Ghub :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following GitHub Flow
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAMMAS-SALEEM commented:
Hey @helina42, @Adams-Ghub I hope you guys are doing well 😊

Well, I am done with my review and gave you a possible solution to your problem. I have also sent you message in your slack so if you need any kind of assistance then you can reach out to me without any hesitation 👍

helina42 committed:
Add styles to popup after review

mahbubul14 reviewed (approved):
# Project Approved :trophy: :tada: 

Hi @helina42, 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well-implemented previous suggestions.

### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```
